src/main/java/com/rakovets/course/java/core/practice/oop/classesandobjects/Point.java

```diff
@@ -0,0 +1,27 @@
+package com.rakovets.course.java.core.practice.oop.classesandobjects;
+
+public class Point {
+    private double x;
+    private double y;
+
+    Point (double x, double y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public double getX() {
+        return this.x;
+    }
+    public void setX(double x) {
```

Comment:
You need to add blank line between two methods: 
```suggestion
    }
    
    public void setX(double x) {
```